egs/zeroth_korean/s5/path.sh

```diff
@@ -0,0 +1,6 @@
+export KALDI_ROOT=`pwd`/../../..
+[ -f $KALDI_ROOT/tools/env.sh ] && . $KALDI_ROOT/tools/env.sh
+export PATH=$PWD/utils/:$KALDI_ROOT/tools/openfst/bin:$PWD:$PATH
+[ ! -f $KALDI_ROOT/tools/config/common_path.sh ] && echo >&2 "The standard file $KALDI_ROOT/tools/config/common_path.sh is not present -> Exit!" && exit 1
+. $KALDI_ROOT/tools/config/common_path.sh
+export LC_ALL=ko_KR.UTF-8
```

Comment:
This is a problem.  LC_ALL should *always* be set to C.  Most people may not even have the Korean locale installed.
Why is this even needed?
If you have scripts that depend on the locale, you should change them so they don't.  With python3, it's possible to rework the I/O so that it treats input streams using a certain encoder/decoder (such as "utf-8").

Reply:
OK. @jty016 will think about this.

Reply:
I removed this dependency by using morfessor's option -e